lib/reek/cli/command.rb

```diff
@@ -1,16 +0,0 @@
-module Reek
-  module CLI
-    #
-    # Base class for all commands
-    #
-    class Command
```

Comment:
I removed the inheritance and replaced it with a mixin below since a) the inheritance lead to spaghetti code pretty fast (e.g. jumping back and forth between parent and kid to follow the flow) and b) was wrong in the first place since it violated the Liskov substitution principle big time.